src/matrix-utils.ts

```diff
@@ -340,14 +341,6 @@ export async function createRoom(
 
   logger.log(`Creating group call in ${result.room_id}`);
```

Comment:
This log line doesn't seem relevant anymore, though it's still useful to log the created room ID I guess